gdx/src/com/badlogic/gdx/graphics/g3d/particles/ParticleChannels.java

```diff
@@ -83,6 +83,20 @@ public void init (FloatChannel channel) {
 		}
 	}
 
+	public static class Scale3Initializer implements ChannelInitializer<FloatChannel> {
```

Comment:
Okay, I fixed the naming and also found that the channel's width was still set to 3 (reduced it to 2).
Thank you for your time by the way! It's appreciated!